odata-protocol/11 Data Service Requests.md

```diff
@@ -426,15 +426,16 @@ application of expand options, expressed as a semicolon-separated list
 of system query options, enclosed in parentheses, see
 [OData-URL](#ODataURL).
 
-Allowed system query options are [`$filter`](#SystemQueryOptionfilter),
+Allowed system query options are
 [`$select`](#SystemQueryOptionselect),
+`$expand`, and
```

Comment:
Is this "and" helpful? Why not a plain comma-separated list?

Reply:
The first three are for all navigation properties (added now)